p5js/02_KNNImage/sketch.js

```diff
@@ -1,24 +1,21 @@
-/*
-===
-KNN Image Classifier Demo
-===
-*/
+/* ===
+ML5 Example
+02_KNN_Image
+KNN Image Classifier example with p5.js
+=== */
 
 let knn;
 let video;
 
-function preload() {
-  // Initialize the KNN method.
-  knn = new ml5.KNNImageClassifier(modelLoaded, 2, 1);
-}
-
 function setup() {
-  createCanvas(320, 240).parent('canvasContainer');
-  video = createCapture(VIDEO);
-  background(0);
-  video.size(227, 227);
-  video.hide();
+  noCanvas();
+  video = createCapture(VIDEO).parent('videoContainer');
+  // Create a KNN Image Classifier
+  knn = new ml5.KNNImageClassifier(2, 1, modelLoaded, video.elt);
+  createButtons();
+}
 
+function createButtons() {
   // Train buttons
   buttonA = select('#buttonA');
   buttonA.mousePressed(() => {
```

Comment:
This is likely to change very soon but the p5 examples don't use ES6 `=>` yet so to stay friendly let's use ES5 `function`. Anonymous is ok I think. For the "plain JS examples" I'm ok with `=>`. What do you think?

Reply:
Yep! Agree. Let's keep anonymous functions in this case.